internal/qtest/test_helper.go

```diff
@@ -48,6 +49,12 @@ func TestDB(t *testing.T) (database.Database, func()) {
 	}
 }
 
+// TestConfig creates a configuration for test instance of the AutograderService
+func TestConfig(t *testing.T) *config.Config {
```

Comment:
The `t` argument is not used yet. Maybe it will, but as is it is not necessary.

Also the doc comment should say: `... returns a configuration for testing the AutograderService.`